### DISCUSSION TITLE
Tuple with unspecified relations

### DISCUSSION THREAD

**Discussion by costap** [ory/keto#604] [Q&A] [answered]
Hi 

Does Keto ( or has plans ti) support tuples with unspecified relations? Like `#...` in the paper?

To express things like parent folders of a doc? e.g. `doc:readme#parent@folder:A#...      doc:readmeis infolder:A`

Thanks
Pedro

**Comment by zepatrik** [comment, marked as answer]
Not yet, but there will be support soon. We wanted to use `_` as this special relation, as it is more explicit than just an empty string.
